nixos/modules/programs/firejail.nix

```diff
@@ -25,12 +35,38 @@ in {
     enable = mkEnableOption "firejail";
 
     wrappedBinaries = mkOption {
-      type = types.attrsOf types.path;
+      type = types.attrsOf (types.either types.path (types.submodule {
+        options = {
+          executable = mkOption {
+            type = types.path;
+            description = "Executable to run sandboxed";
+            example = literalExample "''${lib.getBin pkgs.firefox}/bin/firefox";
+          };
+          profile = mkOption {
+            type = types.nullOr types.path;
+            default = null;
+            description = "Profile to use";
+            example = literalExample "''${pkgs.firejail}/etc/firejail/firefox.profile";
+          };
```

Comment:
Would it make sense to have `profile` as a string instead?

Reply:
this would prevent `profile = pkgs.writeText "name" "content"`

Reply:
Does this answer satisfy you?

Reply:
Sorry forgot this PR, yes that was a use-case I didn't consider. :)